test/SerialIOTest.cpp

```diff
@@ -83,6 +83,54 @@ TEST_CASE( "multi_series_test", "[serial]" )
     allSeries.clear();
 }
 
+TEST_CASE( "multiple_series_handles_test", "[serial]" )
+{
+    /*
+     * First test: No premature flushes through destructor when another copy
+     * is still around
+     */
+    {
+        std::unique_ptr< openPMD::Series > series_ptr;
+        {
+            openPMD::Series series(
+                "sample%T.json", openPMD::AccessType::CREATE );
+            series_ptr = std::unique_ptr< openPMD::Series >(
+                new openPMD::Series( series ) );
+            /*
+             * we have two handles for the same Series instance now:
+             * series and series_ptr
+             * series goes out of scope before series_ptr
+             * destructor should only do a flush after both of them go out of
+             * scope, but it will currently run directly after this comment
+             * since no iteration has been written yet, an error will be thrown
+             */
+        }
+        series_ptr->iterations[ 0 ].meshes[ "E" ][ "x" ].makeEmpty< int >( 1 );
+    }
+    /*
+     * Second test: A Series handle should remain accessible even if the
+     * original handle is destroyed
+     */
+    {
+        std::unique_ptr< openPMD::Series > series_ptr;
+        {
+            openPMD::Series series(
+                "sample%T.json", openPMD::AccessType::CREATE );
+            series_ptr = std::unique_ptr< openPMD::Series >(
+                new openPMD::Series( series ) );
```

Comment:
```suggestion
just for modern C++, `new`-free style:
```suggestion
            series_ptr = std::make_unique< openPMD::Series >( series );
```
```